packages/gatsby/src/bootstrap/load-plugins/index.ts

```diff
@@ -11,15 +11,16 @@ import {
   handleMultipleReplaceRenderers,
   ExportType,
   ICurrentAPIs,
-  validateConfigPluginsOptions,
 } from "./validate"
 import {
   IPluginInfo,
   IFlattenedPlugin,
   ISiteConfig,
   IRawSiteConfig,
 } from "./types"
+import { validateConfigPluginsOptions } from "gatsby-plugin-utils"
 import { IPluginRefObject, PluginRef } from "gatsby-plugin-utils/dist/types"
+import reporter from "gatsby-cli/lib/reporter"
```

Comment:
Rather than importing reporter, it's better to pass it in. I'd add it to loadPlugins, and then pass it in from the bootstrap service where it's used.